playbooks/infra/deploy-vm-bastion-libvirt.yml

```diff
@@ -207,7 +207,7 @@
       ansible.builtin.import_role:
         name: redhatci.ocp.create_vms
       vars:
-        additional_virt_install_options: "--events on_reboot=restart --disk {{ cloud_init_iso_path }},device=cdrom --install no_install=yes --wait=0 \\"
+        additional_virt_install_options: "--events on_reboot=restart --disk {{ cloud_init_iso_path }},device=cdrom --install no_install=yes --wait=0"
```

Comment:
It's was fixed by Shai https://github.com/openshift-kni/eco-ci-cd/pull/210/files, not sure it's relevant in this PR